tests/src/test/java/alluxio/client/fs/FileOutStreamAsyncWriteIntegrationTest.java

```diff
@@ -113,16 +165,18 @@ public void asyncWritePersistWithNoAutoPersist() throws Exception {
 
     mFileSystem.persist(filePath);
     CommonUtils.sleepMs(1);
-    checkPersistStateAndWaitForPersist(filePath, 2);
+    IntegrationTestUtils
+        .checkPersistStateAndWaitForPersist(sLocalAlluxioClusterResource, mFileSystem, filePath, 2);
   }
 
   @Test
   public void asyncWriteWithPersistWaitTime() throws Exception {
     AlluxioURI filePath = new AlluxioURI(PathUtils.uniqPath());
-    createTwoBytesFile(filePath, 2000);
+    createTwoBytesFile(mFileSystem, filePath, 2000);
 
-    CommonUtils.sleepMs(1000);
-    checkPersistStateAndWaitForPersist(filePath, 2);
+    CommonUtils.sleepMs(500);
```

Comment:
if we shorten all the intervals for the cluster, can we shorten this?

Or, does the last line do the wait anyways? If so, does this line do anything?

Reply:
Not sure why the original wait time was 1000ms. I just dropped to 500 because all the heartbeat times are slightly less than that. I don't know what the lower bound is. I think it can just be removed?

Reply:
i removed this line